### PR TITLE
Fix some quick-builds + french translation of all upgrades

### PR DESCRIPTION
**Pull request by Gan0n29**

### CODE REVIEW

**Comment by sirjorj** [comment]
Shadow Sq Vet still has a Cannon instead of a Turret.

**Comment by Gan0n29** [comment]
Hi, 

Can you pull in your repository those changes ?

Thanks.

**Review by danrs — APPROVED** [review]
Approving these changes. I'll give other maintainers a day or so to add input before merging

**Pull request merged**
